src/main/resources/modules/dermatitis.json

```diff
@@ -630,8 +579,8 @@
             "condition_type": "Active Medication",
             "codes": [
               {
-                "system": "RxNorm",
-                "code": "835900",
+                "system": "SNOMED-CT",
```

Comment:
This looks like a reversion of #277 ... I guess we didn't merge those module changes back into the module builder. I can almost guarantee this isn't the only time that's going to happen so maybe we want a less manual process to keep them in sync.

Reply:
Weird. I didn't even notice. We can probably hook something into a successful build on master to push to the module builder, in the same way we do the graphviz images to the gh-pages.